File: exercises/small_problems_101/easy_8/double_char_1.rb
def repeater(string)
  result = ''
  string.chars.each do |letter|
    result << letter * 2
  end
  result
end

puts repeater('Hello') == "HHeelllloo"
puts repeater("Good job!") == "GGoooodd  jjoobb!!"
puts repeater('') == ''

File: frontend/javascript/exercises/easy_5/double_char_1.js
function repeater(sentence) {
  var result = '';
  for (var i = 0; i < sentence.length; i++) {
    result += sentence[i] + sentence[i];
  }
  return result;
}

console.log(repeater('Hello'));     // "HHeelllloo"
console.log(repeater("Good job!")); // "GGoooodd  jjoobb!!"
console.log(repeater(''));          // ''